cases/bueno.ts

```diff
@@ -15,16 +25,90 @@ const dataType = object({
   }),
 });
 
-export class BuenoCase extends Case implements Case {
-  name = 'bueno';
+const dataTypeStrict = objectExact({
+  number: number,
+  negNumber: number,
+  maxNumber: number,
+  string: string,
+  longString: string,
+  boolean: boolean,
+  deeplyNested: objectExact({
+    foo: string,
+    num: number,
+    bool: boolean,
+  }),
+});
+
+const dataTypeLoose = objectInexact({
+  number: number,
+  negNumber: number,
+  maxNumber: number,
+  string: string,
+  longString: string,
+  boolean: boolean,
+  deeplyNested: objectInexact({
+    foo: string,
+    num: number,
+    bool: boolean,
+  }),
+});
+
+addCase(
+  'bueno',
+  'parseSafe',
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
What do you think of using a shared type, e.g.:

```
type UnknownData = any
```

And then just import it everywhere. This will give us a better name and context for why it's `any`, and also avoid the `eslint-disable-next-line` eye sores.

Reply:
Oh of course that makes absolute sense. Will fix that